Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -49,6 +49,57 @@ def get_all_installed_packs(client: demisto_client, unremovable_packs: list):
         return None
 
 
+def uninstall_all_packs_one_by_one(client: demisto_client, hostname, unremovable_packs: list):
+    """ Lists all installed packs and uninstalling them.
+    Args:
+        client (demisto_client): The client to connect to.
+        hostname (str): cloud hostname
+        unremovable_packs: list of packs that can't be uninstalled.
+
+    Returns (bool):
+        A flag that indicates if the operation succeeded or not.
+    """
+    packs_to_uninstall = get_all_installed_packs(client, unremovable_packs)
+    logging.info(f'Starting to search and uninstall packs in server: {hostname}, packs count to '
+                 f'uninstall: {len(packs_to_uninstall)}')
+    uninstalled_count = 0
+    failed_to_uninstall = []
+    if packs_to_uninstall:
+        for pack_to_uninstall in packs_to_uninstall:
+            logging.info(f"Attempting to uninstall a pack: {pack_to_uninstall}")
```

Comment:
What happens if we uninstall pack's dependencies before the pack itself?

Reply:
@daryakoval according to Dudu/Anar it will work regardless of dependant packs being uninstalled first